.rubocop.yml

```diff
@@ -7,6 +7,10 @@ AllCops:
   Exclude:
     - spec/support/spec_helper.rb
     - spec/rails_helper.rb
+    - db/**
+    - docs/**
+    - config/**/*
+    - bin/**
```

Comment:
Found this at [Rubocop Config Documentation](http://rubocop.readthedocs.io/en/latest/configuration/):

> Note: The pattern config/** will match any file recursively under config, but this pattern style is deprecated and should be replaced by config/**/*

I didn't know it was deprecated, so I'll fix it as soon as possible.